src/index.tsx

```diff
@@ -14,6 +14,11 @@ const queryClient = new QueryClient({
     queries: {
       refetchOnWindowFocus: false,
       retry: false,
+      useErrorBoundary: (error: any) => {
+        // Automatically throw on 5xx errors. Components that want to handle
+        // errors should set `useErrorBoundary: false` in their queries.
+        return error.response?.status >= 500
```

Comment:
Are we sure that every query is covered? In a sense, I see status queries are disabled, which we don't want to crash the app. But could for example one query on account page "crash" the page to show the error boundary?

Reply:
Yeah some 5xx failures could hit page-level error boundary